Architecture/TodoApp/android/core/src/test/kotlin/com/novoda/todoapp/statistics/presenter/StatisticsPresenterTest.kt

```diff
@@ -20,12 +22,16 @@ class StatisticsPresenterTest {
 
     var displayer: StatisticsDisplayer = Mockito.mock(StatisticsDisplayer::class.java)
 
-    var presenter = StatisticsPresenter(service, displayer)
+    var navDrawerDisplayer: NavDrawerDisplayer = Mockito.mock(NavDrawerDisplayer::class.java)
```

Comment:
If you can, I don't know how! In Kotlin everything needs to be assigned a default value even if it's a `var`. You can use `lateinit` to get around this, and the code will compile, but it doesn't actually initialise the mock properly if you do it this way.